pandas/tests/frame/methods/test_replace.py

```diff
@@ -661,6 +661,15 @@ def test_replace_simple_nested_dict_with_nonexistent_value(self):
         result = df.replace({"col": {-1: "-", 1: "a", 4: "b"}})
         tm.assert_frame_equal(expected, result)
 
+    @pytest.mark.parametrize("dtype", [object])
```

Comment:
This only has one param so it can be in-lined (same as below)

Reply:
Alright done. Thank you for the super-fast response.